e2e-tests/portal/cypress/integration/journeys/admin/fixtures/index.js

```diff
@@ -0,0 +1,93 @@
+const MOCK_USERS = require('../../../../fixtures/users');
+const CONSTANTS = require('../../../../fixtures/constants');
+
+const { BANK1_MAKER1 } = MOCK_USERS;
+
+const BASE_DEAL = {
+  bank: { id: BANK1_MAKER1.bank.id },
+  additionalRefName: 'Mock',
+  exporter: {
+    companyName: 'Mock company',
+  },
+};
+
+const BSS_DEAL_BASE = {
+  ...BASE_DEAL,
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS,
+  submissionType: CONSTANTS.DEALS.SUBMISSION_TYPE.AIN,
+};
+
+const GEF_DEAL_BASE = {
+  ...BASE_DEAL,
+  dealType: CONSTANTS.DEALS.DEAL_TYPE.GEF,
+};
+
+const BSS_DEAL_DRAFT = {
+  ...BSS_DEAL_BASE,
+  status: CONSTANTS.DEALS.DEAL_STATUS.DRAFT,
+  bankInternalRefName: 'Draft BSS',
+};
+
+const BSS_DEAL_READY_FOR_CHECK = {
+  ...BSS_DEAL_BASE,
+  status: CONSTANTS.DEALS.DEAL_STATUS.READY_FOR_APPROVAL,
+  bankInternalRefName: 'Ready Check BSS',
+};
+
+const BSS_DEAL_AIN = {
+  ...BSS_DEAL_BASE,
+  status: CONSTANTS.DEALS.DEAL_STATUS.DRAFT,
+  bankInternalRefName: 'AIN BSS',
+};
+
+const BSS_DEAL_MIA = {
+  ...BSS_DEAL_BASE,
+  submissionType: CONSTANTS.DEALS.SUBMISSION_TYPE.MIA,
+  bankInternalRefName: 'MIA BSS',
+};
+
+const GEF_DEAL_DRAFT = {
```

Comment:
Are we using all these variables? If not, can we add only the things that we need?

Reply:
done